cardano-db-sync/src/Cardano/DbSync/Tracing/ToObjectOrphans.hs

```diff
@@ -6,17 +6,14 @@
 
 module Cardano.DbSync.Tracing.ToObjectOrphans () where
 
-import           Data.Text (Text, pack)
+import           Data.Text (Text)
```

Comment:
Pack is such a generic name, and the rest of this project uses in qualified, so please switch this to use qualified. Eg:
````
import           Data.Text (Text)
import qualified Data.Text as Text
```
and then use `Text.pack` to match the code in the rest of this project.

Reply:
`pack` is not needed anymore. But, you are right, a qualified import of Text is preferable.